ci/vm.sh

```diff
@@ -353,6 +361,20 @@ do_test() {
 	rm -f uploads/*
 
 	color=34
+	colorize $color "== Running Boot & Phone-Home Test =="
+	if [[ $arch == aarch64 ]]; then
+		# aarch64 doesn't work, idk why
```

Comment:
Please add a TODO with an issue URL to track this problem.

Reply:
done